### PR TITLE
Skinsuits

### PR DESCRIPTION
**Pull request by covertcorvid**
## About The Pull Request
Adds pressure-protective suits to players' survival boxes to help survive ship damage.

**Protections**
Suits protect fully against low pressure using mechanical compression rather than air pressure.
Suits partially protect against cold - they are thicker than regular uniforms but not as good as a firesuit or armor.
Suits protect partially against chemical, gas, and bio. The material is, again, thicker than normal uniforms, but not airtight or waterproof.
Helmets are typical bubble helmets, so they offer the same protection against bio, gas, and liquid permeation as other space helmets.
**Other**
Suits are normal-sized - they are not stiff, so they can be folded.

## Why It's Good For The Game
People will have a better chance of living through ship damage, which will make their rounds more enjoyable. These are not spacesuits though, so ship damage still presents danger to the crew, as it should.

In-universe, it makes sense to try to protect your personnel that you invested years of training into. And if your people stay alive, they can repair the hardware assets better.

## Changelog
:cl:
add: Added emergency "skinsuits" and helmets to game
add: Added skinsuits and helmets to starter gear
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -408,3 +408,27 @@ Contains:
 	armor = list(melee = 60, bullet = 40, laser = 40, energy = 50, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
 	strip_delay = 130
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+
```
Comment (Karmic-Skink) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Please move all of this into the NSV13 clothing files as we try not to cause potential conflicts from upstream, ie /nsv13/code/modules/custom_clothes.dm

File: code/game/objects/items/storage/boxes.dm [hunk 2]
```diff
@@ -126,6 +126,8 @@
 	new /obj/item/clothing/mask/breath(src)
 	new /obj/item/tank/internals/emergency_oxygen(src)
 	new /obj/item/reagent_containers/hypospray/medipen(src)
+	new /obj/item/clothing/head/helmet/space/skinsuit(src)
```
Comment (Karmic-Skink) [inline code/game/objects/items/storage/boxes.dm]: Please add an //NSV13 content tag here for easy merging with upstream

File: code/__DEFINES/atmospherics.dm [hunk 3]
```diff
@@ -103,6 +103,9 @@
 #define FIRE_HELM_MIN_TEMP_PROTECT			60		//Cold protection for fire helmets
 #define FIRE_HELM_MAX_TEMP_PROTECT			30000	//for fire helmet quality items (red and white hardhats)
 
+#define EMERGENCY_HELM_MIN_TEMP_PROTECT		2.0		//The helmet is pressurized with air from the oxygen tank. If they don't take damage from that they won't take damage here
```
Comment (Karmic-Skink) [inline code/__DEFINES/atmospherics.dm]: Please add an //NSV13 content tag here for easy merging with upstream

File: code/game/objects/items/storage/boxes.dm [hunk 4]
```diff
@@ -125,6 +125,9 @@
 /obj/item/storage/box/survival/PopulateContents()
 	new /obj/item/clothing/mask/breath(src)
 	new /obj/item/reagent_containers/hypospray/medipen(src)
+	//NSV13 items
+	new /obj/item/clothing/head/helmet/space/skinsuit(src)
+	new /obj/item/clothing/suit/space/skinsuit(src)
```
Comment (Jalleo) [inline code/game/objects/items/storage/boxes.dm]: please add a //NSV13 END OF EXTRA ITEMS HERE

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 5]
```diff
@@ -393,3 +393,37 @@ Contains:
 		torn = TRUE
 		playsound(loc, 'sound/weapons/slashmiss.ogg', 50, 1)
 		playsound(loc, 'sound/effects/refill.ogg', 50, 1)
+
```
Comment (Jalleo) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: You must move this content into the location that Karmic said. Or provide a reason as to why you are not moving it.

File: nsv13/code/modules/clothing/custom_clothes.dm [hunk 6]
```diff
@@ -190,6 +190,34 @@
 	icon_state = "pilot"
 	armor = list("melee" = 15, "bullet" = 0, "laser" = 0, "energy" = 5, "bomb" = 10, "bio" = 0, "rad" = 5, "fire" = 30, "acid" = 5)
 
+/obj/item/clothing/head/helmet/space/skinsuit
```
Comment (Jalleo) [inline nsv13/code/modules/clothing/custom_clothes.dm]: Ditto.

**Pull request merged**